tdrs-backend/tdpservice/settings/common.py

```diff
@@ -86,7 +87,9 @@ class Common(Configuration):
 
     # Email Server
     EMAIL_BACKEND = "django.core.mail.backends.smtp.EmailBackend"
-
+    EMAIL_HOST = "smtp.ees.hhs.gov"
+    EMAIL_HOST_USER = "admin@hhs.gov"
```

Comment:
@ADPennington @n0remac 
Do we want this as the email sender? Perhaps something like `tdp_admin@acf.hhs.gov` instead?

Reply:
Per syncing with Alex on 9/30, we'd like to change this to `no-reply@tanfdata.acf.hhs.gov`